Extractor/NodeSources/NodeSetSource.cs

```diff
@@ -362,7 +412,7 @@ public void BuildNodes(IEnumerable<NodeId> rootNodes, bool isFullBrowse)
                 isFullBrowse);
         }
 
-        private void GetRelationshipData()
+        private void GetRelationshipData(bool getPropertyReferences, HashSet<NodeId> additionalKnownNodes, List<(NodeId, IReference)> newNodes)
```

Comment:
Minor: Method name starts with get despite returning void, and no obvious out parameters